cetmix_tower_server/models/res_partner.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    server_ids = fields.One2many(
+        "cx.tower.server",
+        "partner_id",
+        string="Servers",
+        groups="cetmix_tower_server.group_user",
+    )
+
+    server_count = fields.Integer(
+        "Number of Servers",
+        compute="_compute_server_count",
+        store=True,
```

Comment:
Once again: do we need to store this value?  Stored valued are computed using `sudo` until Odoo 18.0
This may lead to the wrong number of servers shown to users who don't have access to all servers of the partner.